server/xpub-controller/notification.js

```diff
@@ -83,6 +80,22 @@ class Notification {
       })
     return sent
   }
+
+  async sendDashboardEmail(people) {
+    return this._sendEmail(
+      people,
+      'templates/dashboard-email-html.pug',
+      'templates/dashboard-email-text.pug',
+    )
+  }
+
+  async sendFinalSubmissionEmail(people) {
+    return this._sendEmail(
+      people,
+      'templates/final-submission-email-html.pug',
+      'templates/final-submission-email-text.pug',
+    )
+  }
```

Comment:
I'm unable to test this easily on the review environment to ensure the email is being sent. And there's no unit test for `notification.js`.

I think it's worth adding a test to the resolver for this functionality, since we assert on emails within the resolver tests, for example, [here](https://github.com/elifesciences/elife-xpub/blob/develop/server/xpub-server/entities/manuscript/resolvers/submitManuscript.test.js#L192).

We moved the logic out of the resolvers recently, but we kept the tests. Eventually I'd like to rename the tests within the resolvers to be called "functional" tests, and I think it's still worth maintaining/adding to these tests until then.

Reply:
#1371 is to add the `notification.js` tests.

Reply:
Until #1371 is done, we'll want to stick with our existing methods of testing emails.